csharp/sources/Valkey.Glide/GlideString.cs

```diff
@@ -20,13 +20,41 @@ public static class GlideStringExtensions
     /// <returns>A <see cref="GlideString" />.</returns>
     public static GlideString ToGlideString(this string @string) => new(@string);
 
+    /// <summary>
+    /// Convert a <paramref name="int"/> to a <see cref="GlideString" />.
+    /// </summary>
+    /// <param name="int">A <see langword="int" /> to convert.</param>
+    /// <returns>A <see cref="GlideString" />.</returns>
+    public static GlideString ToGlideString(this int @int) => new(@int.ToString());
+
+    /// <summary>
+    /// Convert a <paramref name="long"/> to a <see cref="GlideString" />.
+    /// </summary>
+    /// <param name="long">A <see langword="long" /> to convert.</param>
+    /// <returns>A <see cref="GlideString" />.</returns>
+    public static GlideString ToGlideString(this long @long) => new(@long.ToString());
+
     /// <summary>
     /// Convert a <paramref name="bytes"/> to a <see cref="GlideString" />.
     /// </summary>
     /// <param name="bytes">A <see langword="byte[]" /> to convert.</param>
     /// <returns>A <see cref="GlideString" />.</returns>
     public static GlideString ToGlideString(this byte[] bytes) => new(bytes);
 
+    /// <summary>
+    /// Convert a <paramref name="key"/> to a <see cref="GlideString" />.
+    /// </summary>
+    /// <param name="key">A <see langword="ValkeyKey" /> to convert.</param>
+    /// <returns>A <see cref="GlideString" />.</returns>
+    public static GlideString ToGlideString(this ValkeyKey key) => new(key.ToString());
```

Comment:
Don't convert thru `ToString()` - key may be binary (value too)
Copy & adopt changes from https://github.com/Bit-Quill/valkey-glide/commit/97b2a04a4e86c219b7f547677274f363c29c8395#diff-bf81780bd679b5565ad7723d2eb350bf0c3b9ea6b40ba6e9fb5061624b884a96